content/applications/general/email_communication/email_servers_outbound.rst

```diff
@@ -205,6 +214,8 @@ When an email is sent from Odoo, the following sequence is used to choose the ou
    higher the priority is). Failing to do so, the first server is determined by the servers' names,
    using alphabetical order.
 
+- If there is no mail server, Odoo will rely on the :ref:`system parameter <email-outbound-default-from-filtering>` value.
+
```

Comment:
Slight correction to avoid future tense, update the anchor that is used, and break the line before 100 chars :)
```suggestion
- If there is no mail server, Odoo relies on the :ref:`system parameter <email-outbound-default>`
  value.
  
```